ayon_server/graphql/resolvers/users.py

```diff
@@ -64,13 +64,20 @@ async def get_users(
         sql_conditions.append(f"users.name IN {SQLTool.array(names)}")
 
     if project_name is not None:
+        if not user.is_manager:
```

Comment:
Is `is_admin` the same as `is_manager`?

Reply:
is_admin implies is_manager, so if the user is admin or a service, it evaluates as true as well